README.md

```diff
@@ -30,10 +49,16 @@ remotes::install_github("hta-pharma/maicplus")
 
 ## Tutorial
 
-To learn how to use the `maicplus` R package, refer to the [package website (hta-pharma.github.io/maicplus/)](hta-pharma.github.io/maicplus/).
+To learn how to use the `maicplus` R package, refer to the
+[package website (hta-pharma.github.io/maicplus/)](hta-pharma.github.io/maicplus/).
```

Comment:
@gravesti 
Can you do a quick fix here.. adding https://

To learn how to use the `maicplus` R package, refer to the [package website (https://hta-pharma.github.io/maicplus/)](https://hta-pharma.github.io/maicplus/).